include/k4a/k4a.hpp

```diff
@@ -904,8 +1005,22 @@ class transformation
         }
     }
 
+    /** Transforms the depth image into 3 planar images representing X, Y and Z-coordinates of corresponding 3d points.
+     * Throws error on failure.
+     *
+     * \sa k4a_transformation_depth_image_to_point_cloud
+     */
+    image depth_image_to_point_cloud(const image &depth_image, k4a_calibration_type_t camera) const
+    {
+        image xyz_image;
+        depth_image_to_point_cloud(depth_image, camera, &xyz_image);
+        return xyz_image;
+    }
+
 private:
     k4a_transformation_t m_handle;
+    k4a_calibration_camera_t m_color_calibration;
+    k4a_calibration_camera_t m_depth_calibration;
```

Comment:
These are relatively large structs - they include camera intrinsics and extrinsics and some other stuff, none of which we seem to actually be using in this class.

As far as I can tell, we're only retaining these to keep track of the image dimensions - consider just using image dimensions

Reply:
I got it. Thanks,

Reply:
@williampMSFT Which do you think is better?
```cpp
private:
    k4a_transformation_t m_handle;
    std::pair<int32_t, int32_t> m_color_resolution;
    std::pair<int32_t, int32_t> m_depth_resolution;
};
```
```cpp
private:
    k4a_transformation_t m_handle;
    struct resolution {
        int32_t widht;
        int32_t height;
    };
    resolution m_color_resolution;
    resolution m_depth_resolution;
};
```

Reply:
I personally prefer the second one, but I'll leave it up to you :)

Reply:
I implemented with the second one. 👍